### PR TITLE
Chapter 6 Modelling User

### PR DESCRIPTION
**Pull request by Iskandar19**

### CODE REVIEW

**Comment by Iskandar19** [comment]
Ready

File: test/fixtures/users.yml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# Read about fixtures at https://api.rubyonrails.org/classes/ActiveRecord/FixtureSet.html
```
Comment (ghost) [inline test/fixtures/users.yml]: You don't need to do test, please delete this file

File: test/models/user_test.rb [hunk 2]
```diff
@@ -0,0 +1,7 @@
+require "test_helper"
```
Comment (ghost) [inline test/models/user_test.rb]: You don't need to do test, please delete this file

File: app/models/user.rb [hunk 3]
```diff
@@ -0,0 +1,9 @@
+class User < ApplicationRecord
+  before_save {self.email = email.downcase}
+
+  validates :name, presence: true
+  VALID_EMAIL_REGEX = /\A[\w+\-.]+@[a-z\d\-.]+\.[a-z]+\z/i
+  validates :email, presence: true, length: {maximum: 255}, format: {with: VALID_EMAIL_REGEX}
+
+  has_secure_password
```
Comment (ghost) [inline app/models/user.rb:8]: where your validates password

**Review by ghost — COMMENTED** [review]
cmt

**Comment by Iskandar19** [comment]
ready

**Review by ghost — COMMENTED** [review]
merged

**Pull request merged**